airplane-mode.php

```diff
@@ -529,6 +529,38 @@ public function toggle_css() {
 			wp_enqueue_style( 'airplane-mode', plugins_url( '/lib/css/' . $file, __FILE__ ), array(), $vers, 'all' );
 		}
 
+		public function set_mode( $mode = 'on' ) {
+			if ( ! in_array( $mode, array( 'on', 'off' ) ) ) {
+				$mode = 'on';
+			}
+
+			// Update the setting.
+			$return = update_site_option( 'airplane-mode', $mode );
+
+			// Fire action to allow for functions to run on status change.
+			do_action( 'airplane_mode_status_change', $mode );
+
+			return $return;
```

Comment:
Wouldn't this always return `true` only when the option was updated and not and not necessarily whether the value `$mode` is `on` or `off`?

Asking as I've never seen `update_site_option` used like this and just trying to follow the logic.